app/src/main/res/layout/fragment_events.xml

```diff
@@ -100,6 +100,18 @@
                     app:layout_constraintLeft_toLeftOf="parent"
                     android:scrollbars="vertical" />
 
+                <LinearLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/layout_margin_extra_large"
+                    app:layout_constraintTop_toBottomOf="@id/eventIcon"
+                    app:layout_constraintLeft_toLeftOf="parent">
+                    <include
+                        layout="@layout/content_no_internet" />
+                </LinearLayout>
+
+
+
```

Comment:
extra blank lines. Rest looks good

Reply:
It will be removed in spotless XML. #1815